auth-web/src/components/auth/manage-business/AddNameRequestForm.vue

```diff
@@ -129,7 +138,7 @@ export default class AddNameRequestForm extends Vue {
   private isFormValid (): boolean {
     return !!this.nrNumber &&
       (!!this.applicantPhoneNumber || !!this.applicantEmail) &&
-      CommonUtils.validateNameRequestNumber(this.nrNumber)
+      this.isValidNrNumber(this.nrNumber)
```

Comment:
Much better, this way you don't have any side effects if you want to check the validation of a NR